### PR TITLE
Pair work on Microverse portfolio

### PR DESCRIPTION
**Pull request by abreham1222**
Microverse portfolio
1. Added index.html
2.  Added style.css
3. Added readme.md

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -393,3 +395,78 @@ nav {
...
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-left: 90px;
+  margin-right: 90px;
+  margin-top: 70px;
+  padding-top: 90px;
+  padding-right: 30px;
+  color: #172b4d;
+}
```
Comment (keroloussamy) [inline style.css:421]: - [ ] Kindly make your paragraph match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 2]
```diff
@@ -393,3 +395,78 @@ nav {
...
+  margin-top: 20px;
+  margin-left: 15px;
+  width: 460px;
+  border: #d4cfc9;
+  box-sizing: border-box;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 24px;
+}
+
```
Comment (keroloussamy) [inline style.css:442]: - [ ] Kindly make your inputs all appear and take the same width and height as the Figma design. The `message` doesn't appear.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 3]
```diff
@@ -393,3 +395,78 @@ nav {
...
 }
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 900px;
+  background-color: #fff;
+}
```
Comment (keroloussamy) [inline style.css:406]: - [ ] Kindly your page doesn't take the full width, kindly fix it to fill the full width.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @abreham1222, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abreham1222** [comment]
On my page, I see the perfect design
Screenshot 2022-03-22 220539

**Comment by keroloussamy** [comment]
@abreham1222 I double-checked the design and I still can see the issues. Can you check it with a screen width equal to 375?

**Review by Lordkaito — APPROVED** [review]
Hi @abreham1222, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**